lib/ripper-tags/parser.rb

```diff
@@ -65,6 +65,9 @@ def on_const_path_ref(a, b)
   def on_binary(*args)
   end
 
+  def on_unary(*args)
+  end
```

Comment:
This was needed to avoid trying to process a unary method (`-@`) in a hash [in the rails codebase](https://github.com/rails/rails/blob/6b509dd18ca2a132342e9b850c00afc9b2a024b0/activesupport/lib/active_support/number_helper/number_to_human_converter.rb#L6-L7) (triggered since we now process the keys/values in a hash assignment)